src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationLambdaInEnum2.java

```diff
@@ -0,0 +1,68 @@
+/* Config:                                                                          //indent:0 exp:0
+ * This test-input is intended to be checked using following configuration:         //indent:1 exp:1
+ * tabWidth = 8                                                                     //indent:1 exp:1
+ * basicOffset = 4                                                                  //indent:1 exp:1
+ * braceAdjustment = 0                                                              //indent:1 exp:1
+ * caseIndent = 4                                                                   //indent:1 exp:1
+ * throwsIndent = 4                                                                 //indent:1 exp:1
+ * forceStrictCondition = false                                                     //indent:1 exp:1
+ * lineWrappingIndentation = 4                                                      //indent:1 exp:1
+ */                                                                                 //indent:1 exp:1
+
+package com.puppycrawl.tools.checkstyle.checks.indentation.indentation; //indent:0 exp:0
+
+import java.util.function.Function; //indent:0 exp:0
+
+enum InputIndentationLambdaInEnum2 { //indent:0 exp:0
+	enum1( //indent:8 exp:8
+		v -> v, //indent:16 exp:16
```

Comment:
Please add case with `v -> v, //indent:12 exp:12`, to make line wrapping indentation clear (+4)

Reply:
done

Reply:
Resolved.